extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -87,23 +87,27 @@ export class AmpImageSlider extends AMP.BaseElement {
     /** @private {boolean} */
     this.disableHint_ = this.element.hasAttribute('disable-hint');
```

Comment:
Does `this.hint_` have to be a user-provided div? Let's refactor this per our discussed approach. We should customize the hint via CSS only via a user-exposed custom class.

Reply:
Held for P1